scripts/autotools/libmongocrypt/CheckSSL.m4

```diff
@@ -0,0 +1,35 @@
+if test "$PHP_CLIENT_SIDE_ENCRYPTION" != "no"; then
+  AC_MSG_CHECKING([which crypto library to use for libmongocrypt])
+  AC_MSG_RESULT([$PHP_MONGODB_SSL])
+
+  dnl Disable Windows crypto
+  AC_SUBST(MONGOCRYPT_ENABLE_CRYPTO_CNG, 0)
+
+  if test "$PHP_MONGODB_SSL" = "darwin"; then
+    AC_SUBST(MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION, 1)
+    AC_SUBST(MONGOCRYPT_ENABLE_CRYPTO, 1)
+    AC_SUBST(MONGOCRYPT_ENABLE_CRYPTO_OPENSSL, 0)
+    AC_SUBST(MONGOCRYPT_ENABLE_CRYPTO_COMMON_CRYPTO, 1)
+
+    PHP_MONGODB_ADD_SOURCES([src/libmongocrypt/src/], "crypto/commoncrypto.c", $PHP_MONGODB_LIBMONGOCRYPT_CFLAGS)
```

Comment:
I'll elaborate on this in my later comment on `libmongocrypt.m4`, but I think `config.m4` is the best place for all of the `PHP_MONGODB_ADD_SOURCES` and similar calls.